app/main.py

```diff
@@ -1,8 +1,47 @@
+from decimal import Decimal
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
```

Comment:
Place all arguments in the `__init__` on a new line including `self`.

Reply:
not fixed